code/game/antagonist/outsider/ert.dm

```diff
@@ -27,20 +27,4 @@ var/datum/antagonist/ert/ert
 	..()
 	ert = src
 
-/datum/antagonist/ert/greet(var/datum/mind/player)
-	if(!..())
-		return
-	to_chat(player.current, "The Emergency Response Team works for Asset Protection; your job is to protect [current_map.company_name]'s assets. There is a code red alert on [station_name()], you are tasked to go and fix the problem.")
-	to_chat(player.current, "You should first gear up and discuss a plan with your team. More members may be joining, don't move out before you're ready.")
```

Comment:
I'm assuming these will be handled on a per-team basis instead, because this is useful albeit basic info

Reply:
Yeah, ghost spawners have a custom spawn message that you can set.